event-stream-processing/src/node-css-parser.test.ts

```diff
@@ -0,0 +1,149 @@
+import { NodeCssParser } from './node-css-parser';
+
+test('basic', () => {
+  const filter = new NodeCssParser();
+
+  /* eslint-disable */
+
+  // record from fluent-bit looks like this:
```

Comment:
Thanks for taking a look at this.
I was actually wondering if the output I have from Fluent-bit is in the right format.
The original record has the '=>' notation. perhaps its JSON represented as a string..
hopefully it reaches the parser as pure JSON.

Reply:
@TimCsaky , the default format for [stdout ](https://docs.fluentbit.io/manual/pipeline/filters/standard-output) is msgpack. Try changing it to `json_lines`

Reply:
Perfect.. thank you! this Fluent bit really is great.
i've changed to `json_lines` and it's making more sense now.